docs/source/en/main_classes/deepspeed.mdx

```diff
@@ -795,6 +795,39 @@ default value in the following cases:
    the increased data buffers.
 
 
+#### ZeRO-0 Config
+
+I'm listing Stage 0 and 1 last since they are rarely used.
```

Comment:
I'm not sure what this brings to the doc. Let's remove this sentence?

Reply:
Perfect, thank you, Sylvain.